### PR TITLE
fix(kensetsu): close cdp only owner

### PR DESCRIPTION
**Pull request by Alexey-N-Chernyshov**
Fix `Burn others KUSD by calling repay_debt function` issue.

### CODE REVIEW

**Review by vovac12 — APPROVED** [review]

File: pallets/kensetsu/Cargo.toml [hunk 1]
```diff
@@ -42,6 +42,7 @@ assets = { path = "../assets", default-features = false }
 [features]
 default = ["std"]
 std = [
+	"assets/std",
```
Comment (wer1st) [inline pallets/kensetsu/Cargo.toml:45]: Is it necessary if you use `assets` in `dev-dependecies` only?

File: pallets/kensetsu/src/tests.rs [hunk 2]
```diff
@@ -914,6 +914,25 @@ fn test_repay_debt_only_signed_origin() {
...
+            FixedU128::from_float(0.0),
+            balance!(0),
+        );
+        // Alice is CDP owner
+        let cdp_id = create_cdp_for_xor(alice(), balance!(0), balance!(0));
+
+        assert_noop!(
+            KensetsuPallet::close_cdp(bob(), cdp_id),
+            KensetsuError::OperationNotPermitted
+        );
```
Comment (wer1st) [inline pallets/kensetsu/src/tests.rs:932]: And what about good case?

File: pallets/kensetsu/Cargo.toml (hunk 1 above)
Comment (Alexey-N-Chernyshov) [reply pallets/kensetsu/Cargo.toml:45]: Yes, tests require `std` feature on assets.

File: pallets/kensetsu/src/tests.rs (hunk 2 above)
Comment (Alexey-N-Chernyshov) [reply pallets/kensetsu/src/tests.rs:932]: Good cases present further, where `caller` is `alice`.

**Review by wer1st — APPROVED** [review]

**Pull request merged**